sdks/python/apache_beam/typehints/trivial_inference.py

```diff
@@ -641,6 +654,18 @@ def infer_return_type_func(f, input_types, debug=False, depth=0):
       # No-op introduced in 3.11. Without handling this some
       # instructions have functionally > 2 byte size.
       pass
+    elif opname == 'RETURN_CONST':
+      # Introduced in 3.12. Handles returning constants directly
+      # instead of having a LOAD_CONST before a RETURN_VALUE.
+      returns.add(state.const_type(arg))
+      state = None
+    elif opname == 'CALL_INTRINSIC_1':
+      int_op = intrinsic_one_ops.int_one_ops[arg]
+      if debug:
+        print("Executing intrinsic one op", int_op.__name__.upper())
+      int_op(state, arg)
+    elif opname == 'CALL_INTRINSIC_2':
```

Comment:
Should we add a comment sth like: all currently available binary intrinsic ops are not relevant for type-checking, and as i mentioned above, also have some safeguard to revisit this logic if we can detect that the list of intrinsic ops has changed.

Reply:
I'm inclined to think I can remove the intrinsic 2 call block here for now so if there's a failure later we will get the clear "cannot handle operation x" failure. I think this was a holdover from when I was anticipating having to implement most of the new calls to get tests green.